### DISCUSSION TITLE
exit range item without a back button

### DISCUSSION THREAD

**Discussion by UserCodeX** [forntoh/LcdMenu#263] [Q&A]
Hi, thanks for the fantastic library! @forntoh 

I'm using it in a project and encountered an issue with menu navigation. To control the menu, I have three buttons: up, down, and enter.

Here's the problem: when I select a range item, I’m unable to exit it. My expectation was to press "enter" to select the range item, adjust its value with "up" and "down," and then press "enter" again to confirm the value and exit the item, so that I can select a different item.

Do you have any suggestions for handling this behavior or any recommended approach to achieve this?

**Comment by forntoh** [comment]
Thanks for using LcdMenu,

Of course, I am aware of that limitation and I am actively working to release a new version soon with bigger features, if you want to try them out already, just use the latest changes from the master branch.

I am currently writing documentation for the new features, once it's ready, then I will create a release.

See #262
